React/src/types/model/Employee.ts

```diff
@@ -5,13 +5,26 @@ export type Employee = {
   firstName: string;
   lastName: string;
   email: string;
+  password: string;
+  username: string;
   status: string;
   roles: string[];
   phone: string;
-  unavailbilityDescription:string;
+  unavailbilityDescription: string;
+  pesel: string;
 };
 
 export const EmployeeStatus = {
   AKTYWNY: "AKTYWNY",
   NIEAKTYWNY: "NIEAKTYWNY",
+}
+
+export const RoleStatus = {
+  MANAGER: "MANAGER",
+  SALES_REPRESENTATIVE: "SALES_REPRESENTATIVE",
+  SPECIALIST: " SPECIALIST",
+  WAREHOUSE_MAN: "WAREHOUSE_MAN",
+  WAREHOUSE_MANAGER: "WAREHOUSE_MANAGER",
+  FITTER: "FITTER",
+  FOREMAN: "FOREMAN"
 }
```

Comment:
Stanowiska powinny być przetłumaczone.
Do tego pytanie, czy nazwa 'RoleStatus' jest tutaj odpowiednia- to jest rola (stanowisko) a nie status